test/components/form/trash-row.spec.js

```diff
@@ -0,0 +1,102 @@
+import DateTime from '../../../src/components/date-time.vue';
+import FormTrashRow from '../../../src/components/form/trash-row.vue';
+
+import Form from '../../../src/presenters/form';
+import { ago } from '../../../src/util/date-time';
+
+import testData from '../../data';
+import { mockLogin } from '../../util/session';
+import { mount } from '../../util/lifecycle';
+
+const mountComponent = (formData) => mount(FormTrashRow, {
+  propsData: { form: new Form(formData) },
+  requestData: {
+    forms: testData.extendedForms.sorted()
+  }
+});
+
+describe('FormTrashRow', () => {
+  describe('form name and deleted timestamp', () => {
+    beforeEach(mockLogin);
+
+    it('shows the form name if it has one', () => {
+      const formData = { xmlFormId: 'f', name: 'My Form' };
+      mountComponent(formData).get('.form-name').text().should.equal('My Form');
+    });
+
+    it('shows the xmlFormId if the form does not have a name', () => {
+      const formData = { xmlFormId: 'f', name: null };
+      mountComponent(formData).get('.form-name').text().should.equal('f');
+    });
+
+    it('shows the deleted timestamp', () => {
+      const formData = { xmlFormId: 'f', name: null, deletedAt: ago({ days: 15 }).toISO() };
+      mountComponent(formData).get('.deleted-date').text().should.match(/^Deleted .+$/);
+      // Should I try to match the date string exactly?
```

Comment:
get the DateTime component and check iso prop
or just delete this comment (lower priority)